src/dealDashboard/dealVotes/dealVotes.ts

```diff
@@ -74,4 +76,9 @@ export class DealVotes {
   get statusText() {
     return this.everyTextCopy.find(textCopy => textCopy.condition());
   }
+
+  vote(value: boolean) {
+    this.eventAggregator.publish("showMessage", "Vote submitted");
```

Comment:
How it ought to work:

1) upon submission the vote button should show a spinny icon
2) put the vote call in try/catch
3) await deal.vote()
4) stop the spinning
4) if it hasn't crash then issue a nice popup notif "Your vote has been successfully submitted"
5) if it has crashed then a notif "Sorry, an error occurred submitting your vote"